presto-main/src/test/java/com/facebook/presto/type/TestBigintOperators.java

```diff
@@ -256,6 +257,14 @@ public void testOverflowDivide()
         assertInvalidFunction(format("%s / -1", Long.MIN_VALUE), NUMERIC_VALUE_OUT_OF_RANGE);
     }
 
+    @Test
+    public void testIndeterminate()
+    {
+        assertOperator(INDETERMINATE, "cast(1 as bigint)", BOOLEAN, false);
+        assertOperator(INDETERMINATE, "cast(4499999999 as bigint)", BOOLEAN, false);
+        assertOperator(INDETERMINATE, "4499999999", BOOLEAN, false);
```

Comment:
@martint As we discussed before, `INDETERMINATE` could not take `NULL` as input, if we do so, and input type of `INDETERMINATE` must be a boxed type like `Long` which might cause overhead when unboxing.